src/node.cc

```diff
@@ -4114,10 +4114,11 @@ inline void PlatformInit() {
       continue;
     // Anything but EBADF means something is seriously wrong.  We don't
     // have to special-case EINTR, fstat() is not interruptible.
-    // if (errno != EBADF)
-    //   ABORT();
-    if (fd != open("/dev/null", O_RDWR))
+    if (errno != EBADF)
       ABORT();
+    open("/dev/null", O_RDWR);
```

Comment:
Thanks for the feedback. Yes, no doubt. This is not intended to be merged. That's what the WIP (work in progress) and the `in progress` label are meant to indicate. Trying some non-intuitive (and quite possibly ignorant) things to try to figure out why this bit of code seems to be not working (or perhaps the associated test is what is not working) on AIX but working on all the other POSIX systems we test on.